fastplotlib/layouts.py

```diff
@@ -7,6 +7,12 @@
 from wgpu.gui.auto import WgpuCanvas
 
 
+class GridPlotIndexer:
+    def __init__(self, position):
+        self.name: str = None
```

Comment:
move `name` to `Subplot`, get rid of `GridPlotIndexer` class. 

`position` is already an attribute of `Subplot` so don't need to worry about that.